broker/src/test/java/io/camunda/zeebe/broker/exporter/stream/ExporterDirectorPauseTest.java

```diff
@@ -95,6 +96,13 @@ public void canPauseAndResumeWithoutAnyExporter() {
     // given
     activeExporter.startExporterDirector(List.of());
 
+    // when -- exporter is closed
+    // Wait for exporter to properly close before submitting pause/resume jobs.
+    // Needed to prevent flaky test https://github.com/camunda/zeebe/issues/10439
+    Awaitility.await()
+        .alias("Exporter is closed")
+        .until(() -> activeExporter.getDirector().getPhase().join() == ExporterPhase.CLOSED);
+
```

Comment:
Closed because there is nothing to export right?

Reply:
Yep :+1: https://github.com/camunda/zeebe/blob/4ad7942ffa06e2924477048fe0c3b5b37c2411b6/broker/src/main/java/io/camunda/zeebe/broker/exporter/stream/ExporterDirector.java#L328-L346

Reply:
I'd also rather not rely on that, but I didn't find a better way to wait for this condition before pausing/resuming.

Reply:
Might make sense to change this condition https://github.com/camunda/zeebe/blob/4ad7942ffa06e2924477048fe0c3b5b37c2411b6/broker/src/main/java/io/camunda/zeebe/broker/exporter/stream/ExporterDirector.java#L129 and verify isOpened, which will be set onActorCloseRequested

Reply:
Actually, I would expect that this https://github.com/camunda/zeebe/blob/main/broker/src/main/java/io/camunda/zeebe/broker/exporter/stream/ExporterDirector.java#L206 is only done if we really opened (meaning after the starting of the exporters), and this would mean that the check for isOpened will always return false and pause/resume wouldn't fail.

Reply:
That's a good point, thanks! I'll update the PR

Reply:
Whichever way I turn it, some tests fail if I use `isOpened` and set it only after starting exporters. 
Using `actor.isClosed()` allowed to pause exporting while the exporter director actor is in the starting phase. Using `isOpened` instead, tests like `shouldResumeActiveExporter` fail because pausing is requested before the actor is fully started (isOpened = false).

Do you mind leaving this PR as is for now?

Reply:
Ok thanks for trying @oleschoenburg :bow: